.github/workflows/node.js.yml

```diff
@@ -145,14 +145,11 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
-          cache: 'npm'
       - name: run tests
         env: 
           QMATE_PRIVATE_KEY: ${{ secrets.QMATE_PRIVATE_KEY }}
         run: |
-          sudo apt-get update
-          sudo apt-get install -y --only-upgrade openssl
-          sudo apt-get install google-chrome-stable
+          echo "$QMATE_PRIVATE_KEY" > private.key
```

Comment:
@cvakiitho will this persist in pipeline and will it be accessible from outside? Since we remove the key during qmate init, we need to make sure that we do not have any public copies of it

Reply:
removed since it does not affect anything. key is being picked up from env correcly.

Reply:
The VMs are removed after pipeline is finished, so it won't metter - but yeah- this was only a hack to see it's correctly passed, and it made some encoding issues dissapear in jenkinses.